docs/examples/viz_helical_motion.py

```diff
@@ -68,94 +70,123 @@
 directions = np.array([[1, 0, 0]])
 heights = np.array([8])
 arrow_actor = actor.arrow(centers, directions, color_arrow, heights,
-                          resolution=20, tip_length=0.06, tip_radius=0.012,
+                          resolution=50, tip_length=0.06, tip_radius=0.012,
                           shaft_radius=0.005)
+
+# manipulating shading for aesthetics
+material.manifest_standard(arrow_actor, ambient_level=1, diffuse_level=1,
+                           specular_level=1)
 scene.add(arrow_actor)
 
 ###############################################################################
 # Initializing the initial coordinates of the particle
 
 x = initial_velocity*time + 0.5*acc*(time**2)
-y = np.sin(angular_frq*time + phase_angle)
-z = np.cos(angular_frq*time + phase_angle)
+y = radius_path*np.sin(angular_frq*time + phase_angle)
+z = radius_path*np.cos(angular_frq*time + phase_angle)
+pts = np.array([[x, y, z]])
+
 
 ###############################################################################
-# Initializing point actor which will represent the charged particle
+# Initializing the path_actor which will trace the path followed by the
+# charged particle
 
-color_particle = window.colors.red  # color of particle can be manipulated
-pts = np.array([[x, y, z]])
-charge_actor = actor.point(pts, color_particle, point_radius=radius_particle)
-scene.add(charge_actor)
+num_total_steps = 200  # total number of simulation steps
+origin = np.asarray([0, 0, 0], dtype=float)
+position = np.tile(origin, (num_total_steps, 1))
 
-vertices = utils.vertices_from_actor(charge_actor)
-vcolors = utils.colors_from_actor(charge_actor, 'colors')
-no_vertices_per_point = len(vertices)
-initial_vertices = vertices.copy() - \
-    np.repeat(pts, no_vertices_per_point, axis=0)
+v = np.linspace(0, 1, num_total_steps)
+cmap_name = 'spring'  # using colormap to color the path of the particle
+colors = colormap.create_colormap(v, name=cmap_name)
 
+path_actor = actor.line([position], colors, linewidth=3)
+actor.attributes_to_actor(path_actor, position=position,
+                          no_vertices_per_point_divisor=num_total_steps)
+scene.add(path_actor)
+
+###############################################################################
+# Initializing point actor which will represent the charged particle
+
+color_particle = window.colors.yellow  # color of particle can be manipulated
+charge_actor = actor.point(pts, color_particle, point_radius=radius_particle,
+                           phi=32, theta=32)
+actor.attributes_to_actor(charge_actor, pts)
+
+# manipulating shading for aesthetics
+material.manifest_standard(charge_actor, ambient_level=1, diffuse_level=1,
```

Comment:
Is the effect exactly how it was before? If not then create an additional ``manifest_ambience`` function and only enable ambient light.

Reply:
The effect was a bit different, I created `manifest_ambience` accordingly.